node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -46,7 +47,7 @@ class PeerMessageReceiver(
 
         val timeout =
```

Comment:
Can you rename this `timeout` to be someting like `initializationTimeoutChecker` or something like that?  

I that that name is much more representative of whats going on. You should change in `Initializing` too.